middleware.js

```diff
@@ -194,8 +197,16 @@ module.exports = function(compiler, options) {
 
 	// The middleware function
 	function webpackDevMiddleware(req, res, next) {
+		var goNext = function() {
```

Comment:
While you're at it, could you rewrite this to `function goNext`? that's more consistent with the above function.